R/scale-alpha.r

```diff
@@ -19,17 +17,34 @@
 #' (p <- ggplot(mtcars, aes(mpg, cyl)) +
 #'   geom_point(aes(alpha = factor(cyl))))
 #' p + scale_alpha_discrete(range = c(0.4, 0.8))
-scale_alpha <- function(..., range = c(0.1, 1)) {
-  continuous_scale("alpha", "alpha_c", rescale_pal(range), ...)
+scale_alpha <- function(name = waiver(), range = c(0.1, 1), breaks = waiver(),
```

Comment:
One downside is that this would require `continuous_scale` to be defined before `scale_alpha` in order to inspect the arguments.